src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -785,7 +922,6 @@ def forward(
         output_attentions: Optional[bool] = None,
         output_hidden_states: Optional[bool] = None,
         return_dict: Optional[bool] = None,
-        use_hidden_state: Optional[bool] = True,
```

Comment:
Why are we removing this flag?

Reply:
It was previously used to selectively return unmodified (same as CLIP) base model embeddings but I moved this functionality to the OwlViTModel.forward() with the `return_base_image_embeds` argument. OWL-ViT is trained in two stages and this is to keep the behaviour of the base model (CLIP) the same.